content/guides/building/get-source-git.md

```diff
@@ -48,64 +48,75 @@ to the last 10 commits, for example.
 
 <h4>Configure Git on your system:<a name="configure_env"></a></h4>
 
-Before making your first commit on a new system, be <strong>sure</strong> to configure Git. These global settings are stored in your git configuration directory (`~/.git/` or for Haiku: `~config/settings/git/`) and will be appended to <strong>each</strong> commit as your personal information.
+Using `--global` will your personal information in your Git configuration directory (`~/.git/` or for Haiku: `~config/settings/git/`), which will be included in <strong>every</strong> commit.
 
 ```sh
 git config --global user.name "John Doe"
-git config --global user.email "john.doe@developers.com"
+git config --global user.email "john.doe@example.com"
 ```
 
-On macOS, you must set the following option in order to avoid problems with the unicode representation of filenames:
+On macOS, you must set the following option in order to avoid problems with the Unicode representation of filenames:
 
 ```sh
 git config core.precomposeunicode true
 ```
 
 <h4>Setup an account on Gerrit</h4>
 
-<p>Log in to <a href="https://review.haiku-os.org">Gerrit code review</a>. You currently need a <a href="https://github.com">Github</a> account for logging in.</p>
+<p>Log in to <a href="https://review.haiku-os.org">Gerrit code review</a>. For the time being, using a <a href="https://github.com">GitHub account</a> is necessary.</p>
 
 <div class="alert alert-warning">
-Make sure your ssh key is generated with ed25519. RSA keys will not work with Gerrit.
+Make sure your SSH key is generated using <code class="code">ed25519</code>. RSA keys will <b>not</b> work with Gerrit.
 </div>
-<p>Upload your SSH public key in <a href="https://review.haiku-os.org/settings/#SSHKeys">Gerrit SSH keys settings</a> page. If you don't have a key yet, you can generate one using ssh-keygen.</p>
+<p>Upload your SSH public key in the <a href="https://review.haiku-os.org/settings/#SSHKeys">Gerrit SSH Keys settings</a> page. If you don't have a key yet, you can generate one using ssh-keygen.</p>
 <p>If the e-mail address used in your commits does not match the one in your github account, you will need to add and verify it in <a href="https://review.haiku-os.org/settings/#EmailAddresses">Gerrit E-Mail address settings</a>.</p>
 
-<h4>Build Tools:</h4>
+<div class="alert alert-warning">
+You may need to replace the <code class="varname">$USER</code> parameter in the following commands.
+</div>
+<p>This can happen if the username on your Gerrit account does not match the username that you use to log in to your computer. This would be the case in Haiku itself, as the default username is <code class="code">user</code>. You can make your life easier by <a href="https://linux.die.net/man/5/ssh_config">modifying your SSH configuration</a>.</p>
```

Comment:
A link to a man page documenting the whole file is not very helpful. Especially in this page which is supposed to be an "all-in-one" getting started tutorial.

Reply:
Understandable. Do I just give the reader a sample ~/.ssh/config and tell them the directories to put them in? (e.g. `~/.ssh/config` on Linux, `/boot/system/settings/ssh/sshd_config` on Haiku.

Reply:
If you set the login in the URL as we already recommend, I think the ssh-config is not needed. So it seems simpler to just not mention it at all here?